models/app.models.js

```diff
@@ -6,4 +6,33 @@ exports.fetchTopics = ()=>{
     .then((res)=>{
         return res.rows;
     })
+    .catch((err)=>{
+        next(err);
+    })
+};
+
+exports.fetchArticleById = (article_id) => {
+    console.log('Article ID in model:', article_id);
+    return db
+    .query(
+        `SELECT articles.*, comments.article_id FROM articles 
+        LEFT JOIN comments ON articles.article_id = comments.article_id
+        WHERE articles.article_id = $1
+        GROUP BY articles.article_id, comments.article_id;`,
+        [article_id]
+    )
```

Comment:
For this ticket you don't need to include the comments, there is a later ticket where you will get comments separately for a specific article id on a different endpoint. There is also a separate ticket to get a comment count on an article, but you don't need it yet 👍